.github/workflows/python.yml

```diff
@@ -0,0 +1,65 @@
+name: Build and Distribute PhotonLibPy
+
+permissions:
+  id-token: write  # IMPORTANT: this permission is mandatory for trusted publishing
+
+on:
+  push:
+    branches: [ master ]
+    tags:
+      - 'v*'
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8  # CHose earliest that robotpy supports
+
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install setuptools wheel
+          pip install robotpy
```

Comment:
This doesn't appear to be a build dependency. Even then you probably don't want to manually install build dependencies either.